### PR TITLE
Linting changes for keeping only one Component for file

### PR DESCRIPTION
**Pull request by VenkateshKatabathula**

### CODE REVIEW

File: src/components/views/visualization/Choropleth.js [hunk 1]
```diff
@@ -1,406 +1,329 @@
...
-  Popup,
-  Rectangle,
-  TileLayer, GeoJSON
-} from 'react-leaflet';
-import 'bootstrap/dist/css/bootstrap.css';
+import React, {Component} from "react";
+import {TopojsonData} from "../../../data/StatesTopojson";
+import {FeatureGroup, GeoJSON, Map, TileLayer} from "react-leaflet";
+import "bootstrap/dist/css/bootstrap.css";
+
```
Comment (akshay2905) [inline src/components/views/visualization/Choropleth.js]: Please import StateTooltip Component. There is an import error as of now.

**Comment by akshay2905** [comment]
@VenkateshKatabathula Thank you helping us fixing this issue. I went through the changes. As some of the changes are too monolithic to review, it was a little difficult to review them. So I checked the pr locally. There is one issue that is throwing an import error. I have highlighted in the comment.

File: src/components/views/visualization/Choropleth.js (hunk 1 above)
Comment (VenkateshKatabathula) [inline src/components/views/visualization/Choropleth.js]: @akshay2905 Thanks I think I missed that

**Comment by VenkateshKatabathula** [comment]
@akshay2905 I have updated that can you please check once

**Pull request merged**